crates/paimon/Cargo.toml

```diff
@@ -35,3 +38,4 @@ serde_bytes = "0.11.15"
 snafu = "0.8.3"
 typed-builder = "^0.18"
 opendal = "0.48"
+tokio = { version = "1.39.2", features = ["full"] }
```

Comment:
It's better to avoid enabling `full` features, please only pick what you want.

Reply:
OK